### DISCUSSION TITLE
scrolling in the editor is much faster while the song is unpaused, compared to paused

### DISCUSSION THREAD

**Discussion by tovc** [ppy/osu#23400] [Q&A]
### Type

Game behaviour

### Bug description

I am using a freescroll mouse; if I use freescroll, then pause/unpause the editor, the speed of scrolling varies greatly. I don't know what the 'correct' behaviour here would be, but I believe the existing behaviour is incorrect.

### Screenshots or videos

[HEADPHONE WARNING](https://youtu.be/bpqmz-jKj4M)

### Version

2023.403.1-lazer

### Logs

[network.log](https://github.com/ppy/osu/files/11180708/network.log)
[performance.log](https://github.com/ppy/osu/files/11180709/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11180710/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11180711/updater.log)
[database.log](https://github.com/ppy/osu/files/11180712/database.log)
[input.log](https://github.com/ppy/osu/files/11180713/input.log)

**Comment by OliBomby** [comment]
I think this is the correct behaviour, mimicking the behaviour in the osu! stable editor.

**Comment by tovc** [comment]
mimicking stable behaviour does not inherently mean that it is correct

**Comment by peppy** [comment]
The intention here is to change the scroll speed for non-precision scroll devices to make it easier to navigate the song while it's playing. For precision scroll devices this multiplier should likely not be applied.

**Comment by peppy** [comment]
As per discussion in the [PR thread](https://github.com/ppy/osu/pull/23366), I don't think this needs to be changed. Will wait for more feedback and move to a discussion for now.
